README.md

```diff
@@ -86,6 +86,8 @@ Commands to build and publish using GDK:
 * gdk component build
 * gdk component publish
 
+Note: The container is intended to be used on a linux amd64 architecture. If you wish to deploy this component onto another architecture please modify the --platform type in the Dockerfile
+
```

Comment:
Please add a sentence + link about the supported platforms for the IoT-Device-Client:
https://github.com/awslabs/aws-iot-device-client#list-of-supported-platforms

and Python as second dependency, v3.10 for Ubuntu 22.04: https://docs.python.org/3.10/library/asyncio-platforms.html